README.md

```diff
@@ -5,167 +5,142 @@
+1. Start the server and the local storage service. In the root directory of Ark, run:
 
-To get the server started on your cluster (as well as the local storage service), execute the following commands in Ark's root directory:
+    ```bash
+    kubectl apply -f examples/common/00-prereqs.yaml
+    kubectl apply -f examples/minio/
+    kubectl apply -f examples/common/10-deployment.yaml
+    ```
 
-```
-kubectl apply -f examples/common/00-prereqs.yaml
-kubectl apply -f examples/minio/
-kubectl apply -f examples/common/10-deployment.yaml
-```
+    NOTE: If you get an error about Config creation, wait for a minute, then run the commands again.
 
-*NOTE: If you encounter an error related to Config creation, wait for a minute and run the command again. (The Config CRD does not always finish registering in time.)*
+1. Deploy the example nginx application:
 
-Now deploy the example nginx app:
-```
-kubectl apply -f examples/nginx-app/base.yaml
-```
+    ```bash
+    kubectl apply -f examples/nginx-app/base.yaml
+    ```
 
-Check to see that both the Ark and nginx deployments have been successfully created:
-```
-kubectl get deployments -l component=ark --namespace=heptio-ark
-kubectl get deployments --namespace=nginx-example
-```
+1. Check to see that both the Ark and nginx deployments are successfully created:
 
-Finally, install the Ark client somehwere in your `$PATH`:
-* [Download a pre-built release][26], or
-* [Build it from scratch][7]
+    ```
+    kubectl get deployments -l component=ark --namespace=heptio-ark
+    kubectl get deployments --namespace=nginx-example
+    ```
 
+### Install client
 
-### 3. Back up and restore
-First, create a backup specifically for any object matching the `app=nginx` label selector:
+For purposes of this example, we recommend that you [download a pre-built release][26].
 
-```
-ark backup create nginx-backup --selector app=nginx
-```
+You can also [build from source][7].
 
-Now you can mimic a disaster with the following:
-```
-kubectl delete namespace nginx-example
-```
-Oh no! The nginx deployment and service are both gone, as you can see (though you may have to wait a minute or two for the namespace be fully cleaned up):
-```
-kubectl get deployments --namespace=nginx-example
-kubectl get services --namespace=nginx-example
-```
-Neither commands should yield any results. However, because Ark has your back(up), you can run this command:
-```
-ark restore create nginx-backup
-```
+Make sure that you install somewhere in your `$PATH`.
 
-To check on the status of the Restore:
-```
-ark restore get
-```
+### Back up
 
-The output should look something like the table below:
-```
-NAME                          BACKUP         STATUS      WARNINGS   ERRORS    CREATED                         SELECTOR
-nginx-backup-20170727200524   nginx-backup   Completed   0          0         2017-07-27 20:05:24 +0000 UTC   <none>
-```
+1. Create a backup for any object that matches the `app=nginx` label selector:
```

Comment:
Maybe we should switch this to backup the namespace? Effect is the same, just maybe clearer. @ncdc @nrb thoughts?

Reply:
SGTM but not required for v0.7.0

Reply:
I don't think it's a bad change, but I also agree that probably not as important for v0.7.0.

In this vein, it may be useful to provide a couple different examples in the docs later - using labels, using a namespace, excluding resources.